examples/sentiment/train_sentiment.py

```diff
@@ -127,8 +127,7 @@ def traverse(model, node, evaluate=None, root=True):
         # leaf node
         word = xp.array([node['node']], np.int32)
         loss = 0
-        x = chainer.Variable(word, volatile=not chainer.config.train)
-        v = model.leaf(x)
+        v = model.leaf(word)
```

Comment:
Don't we have to set `no_backprop_mode` or `enable_backprop_mode` depending on the value of `chainer.config.train`?